src/metrics.rs

```diff
@@ -35,6 +35,8 @@ pub struct Metrics {
     pub processed_notifications: Counter<u64>,
     pub dispatched_notifications: Counter<u64>,
     pub notify_latency: Histogram<u64>,
+    pub spawned_publishing_workers: ObservableGauge<u64>,
+    pub publishing_worker_errors: Counter<u64>,
```

Comment:
I'd prefer if these had the same prefix, but just a style thing

Reply:
Makes sense. I've changed to use `publishing_workers_` prefix.